lib/themes/dosomething/paraneue_dosomething/includes/preprocess.inc

```diff
@@ -84,6 +83,11 @@ function paraneue_dosomething_preprocess_page(&$vars) {
     unset($vars['tabs']);
   }
 
+  // Display edited admin tabs for regional admins.
+  if (dosomething_global_is_regional_admin()) {
+    $vars['tabs'] = paraneue_dosomething_edit_regional_admin_tabs($vars['tabs']);
```

Comment:
Any need to check that `$vars['tabs']` is set before passing it into this function?